Example/Tests/CallSpec.swift

```diff
@@ -68,6 +68,70 @@ class CallSpec: QuickSpec {
             }
 
         }
+        
+        describe("Call with parameters") {
+            let configuration = Faro.Configuration(baseURL: "http://someURL")
+
+            it("should insert http headers into the request") {
+                let params = Parameters(type: Parameters.ParameterType.httpHeader, parameters: ["Accept-Language" : "en-US",
+                                                                                                "Accept-Charset" : "utf-8"])
+                let call = Call(path: "path", method: .GET, rootNode: nil, parameters: params)
+                let request = call.request(withConfiguration: configuration)
+                expect(request?.allHTTPHeaderFields?.keys).to(contain("Accept-Language"))
+                expect(request?.allHTTPHeaderFields?.values).to(contain("utf-8"))
+            }
+            
+            it("should fail to insert http headers that arent strings into the request") {
```

Comment:
❓Ik zie niet echt hoe de request kan falen op deze manier? Je checkt allen maar op wat er in zit en niet echt op iets dat faalt. Niet?